plop/component/index.js

```diff
@@ -49,10 +55,10 @@ module.exports = plop => {
       },
       {
         type: "append",
-        path: "src/published-components.js",
+        path: "webpack/published-ts-components.js",
         pattern: /(\/\/ plop_marker:published-components)/g,
         template:
-          '\t{{properCase componentName}}: "/src/components/{{properCase componentName}}/{{properCase componentName}}.jsx",'
+          '  {{properCase componentName}}: "/src/components/{{properCase componentName}}/{{properCase componentName}}",'
```

Comment:
there is no need in \t anymore?

Reply:
It's a tab there, but for some reason it consists of 2 spaces, while this tab consists of 4 spaces. Don't know why this is so, but I've figured out that it would be easier just to replace it with 2 spaces